QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -85,8 +77,23 @@ private static void setupRollers() {
 
         FloatInput cutoffAuto = mainTuning.getFloat("Trigger Threshold +M", 0.5f);
 
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(3), cutoffAuto), QuasarHelios.autoEjector);
-        BooleanMixing.pumpWhen(QuasarHelios.manualControl, FloatMixing.floatIsAtLeast(Igneous.joystick2.getAxisSource(4), cutoffAuto), QuasarHelios.autoLoader);
+        FloatInput leftTrigger = Igneous.joystick2.getAxisSource(3);
+        FloatInput rightTrigger = Igneous.joystick2.getAxisSource(4);
+
+        BooleanInputPoll rollersMode = Igneous.joystick2.getButtonChannel(5);
```

Comment:
It's holding a bumper while using the triggers. I've seen it work. It might be better if the bumpers toggled between modes instead of having to be held, though that could also be confusing to remember which mode you're in. I am open to suggestions.